src/server/api/routers/profile.ts

```diff
@@ -0,0 +1,28 @@
+import { clerkClient } from "@clerk/nextjs";
+import { TRPCError } from "@trpc/server";
+import { z } from "zod";
+
+import { createTRPCRouter, publicProcedure } from "~/server/api/trpc";
+
+const modelProcedure = publicProcedure
+
+export const profileRouter = createTRPCRouter({
+  getUserByUsername: modelProcedure.input(z.object({
```

Comment:
```suggestion
  show: modelProcedure.input(z.object({
```
`endpoint`名についてですが、`show`の方がSGの命名規則に沿っているかもしれません！
今回こちらの処理では、`user`が一つだけ取得できる処理だと思いますのでcrud操作の命名規則に合わせると`show`が良さそうです🙇